src/utils/fetch-config.ts

```diff
@@ -31,6 +31,18 @@ export async function fetchConfig (): Promise<void> {
     return Promise.reject(new Error('Could not fetch configuration.json'))
   })
 
+  /**
+   * This is a workaround to fix the sbc-common-components that expect their own session keys.
+   * Ref: #6801
+   */
+  const authApiConfig = {
+    VUE_APP_AUTH_ROOT_API: response.data['SBC_CONFIG_AUTH_API_URL'],
+    VUE_APP_STATUS_ROOT_API: response.data['VUE_APP_STATUS_ROOT_API']
```

Comment:
Where is it used? I don't see where VUE_APP_STATUS_ROOT_API is used...

Reply:
Good eye! It is not used by our code. It is used by the SbcHeader and PaySystemAlert common components.

That's why I wrote that comment above this code :)

You don't need to read 6801 but basically it acknowledges that a work-around is needed at the moment.

Reply:
ok then :)